src/pfe/portal/modules/User.js

```diff
@@ -34,13 +34,13 @@ const util = require('util');
  */
 module.exports = class User {
 
-  static async createUser(user_id, userString, workspace, uiSocket) {
-    let user = new User(user_id, userString, workspace, uiSocket);
+  static async createUser(user_id, userString, workspace, uiSocket, k8Client) {
```

Comment:
An alternative to this could be adding `global.codewind.K8S_CLIENT`, I'm not totally happy with passing the client as another argument since it would be the same client for all users (if we had more than one) but I'm also not that happy about adding another global. :-)

Reply:
updated to incl in global obj

Reply:
gonna push the changes in a new PR